File: DPBO-praktikum/1/PHP/Mahasiswa.php
<?php
class Mahasiswa
{
  private $nama;
  private $nim;
  private $prodi;
  private $fakultas;
  private $foto_profil;

  public function __construct($nama, $nim, $prodi, $fakultas, $foto_profil) {
      $this->nama = $nama;
      $this->nim = $nim;
      $this->prodi = $prodi;
      $this->fakultas = $fakultas;
      $this->foto_profil = $foto_profil;
  }

  public function getName() {
      return $this->nama;
  }

  public function getNim() {
      return $this->nim;
  }

  public function getProdi() {
      return $this->prodi;
  }

  public function getFakultas() {
      return $this->fakultas;
  }

  public function getFotoProfil() {
      return $this->foto_profil;
  }

  public function setName($nama) {
      $this->nama = $nama;
  }

  public function setNim($nim) {
      $this->nim = $nim;
  }

  public function setProdi($prodi) {
      $this->prodi = $prodi;
  }

  public function setFakultas($fakultas) {
      $this->fakultas = $fakultas;
  }

  public function setFotoProfil($foto_profil) {
      $this->foto_profil = $foto_profil;
  }
}
?>

File: DPBO-praktikum/1/Java/Mahasiswa.java
/**
 * Mahasiswa
 */
public class Mahasiswa {
    private String nama;
    private String nim;
    private String prodi;
    private String fakultas;

    public Mahasiswa(){

    }
    public Mahasiswa(String nama, String nim, String prodi, String fakultas){
        this.nama=nama;
        this.nim=nim;
        this.prodi=prodi;
        this.fakultas=fakultas;
    }
    public void set_nama(String nama){
        this.nama=nama;
    }
    public String get_nama(){
        return this.nama;
    }
    public void set_nim(String nim){
        this.nim=nim;
    }
    public String get_nim(){
        return this.nim;
    }
    public void set_prodi(String prodi){
        this.prodi=prodi;
    }
    public String get_prodi(){
        return this.prodi;
    }
    public void set_fakultas(String fakultas){
        this.fakultas=fakultas;
    }
    public String get_fakultas(){
        return this.fakultas;
    }

}